src/datamodel/turn/test/test_quest_proposer_role_action_manager.rs

```diff
@@ -1,8 +1,9 @@
-use crate::datamodel::quest::quest_proposer_role_action_manager::*;
+use crate::datamodel::turn::QuestProposerRoleActionManager;
 
 #[test]
 fn test_quest_proposer_role_action_manager_is_correct() {
     //unimplemented yet
     // TODO
+    let _ = QuestProposerRoleActionManager {};
     assert!(true);
```

Comment:
remove this please

Reply:
I plan to leave `assert!(true)` in all unimplemented methods, so that's also a way of detecting that we actually haven't implemented anything. I took this from your integration test.

Reply:
hahaha, it's my fault then :p. I was recently made aware of this characteristic of Rust's tests Methods.

Reply:
Done. I already changed it and put the same `// TODO implement` comment in all of them.